pkgs/tools/misc/geekbench/6.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, linuxPackages
+, ocl-icd
+, vulkan-loader
+}:
+
+stdenv.mkDerivation rec {
+  pname = "geekbench";
+  version = "6.0.1";
+
+  src = fetchurl {
+    url = "https://cdn.geekbench.com/Geekbench-${version}-Linux.tar.gz";
+    hash = "sha256-RrfyB7RvYWkVCbjblLIPOFcZjUR/fJHk1Em1HP74kmY=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [
+    linuxPackages.nvidia_x11
```

Comment:
For support for CUDA, I think it is better to use `addOpenGLRunpath` (it is a misnomer, it also works for non OpenGL drivers). This way, this package doesn't depend directly in a specific `NVIDIA` driver (that may or may not work since this driver is compiled to a specific version of kernel).